plane/source/docs/apms-failsafe-function.rst

```diff
@@ -7,111 +7,85 @@ Plane Failsafe Function
+#. If Throttle Failsafe is setup (ie via trim tab or transmitter switch). Check that it operates correctly by activating it and watching for Short and Long Failsafes to occur.
 
-        |_________________|
-                 |
-              Failsafe
+If you observe this behavior, your RC Failsafe function has been set up correctly. If not, recheck that the parameters above have been set correctly.
 
-**Setup.**
+.. _old_RX:
+
+Older Receivers
+~~~~~~~~~~~~~~~
 
-#. Enable RC failsafe by setting :ref:`THR_FAILSAFE<THR_FAILSAFE>` to 1 (0=Disabled,
-   1=Enabled, 2=Enabled, No Failsafe action undertaken).
-#. First turn on your transmitter and enable the throttle range to
-   extend past -100%, we want to extend the throttle range past its low
-   threshold.
-#. Once this is done, bind with your receiver. This will let your
-   receiver know the lowest possible value for your throttle channel.
-#. Next revert the first change you made to the transmitter to limit the
-   throttle to the original range.
-#. Do the radio calibration using the Mission Planner.
-#. Once the radio calibration is completed, drop the throttle on your
-   transmitter and read what PWM value is being output to the mission
-   planner on that channel.
-#. Turn off the transmitter. You should see the value drop
-   significantly. This will be the PWM value relayed to the autopilot in
-   the event RC link was lost during flight.
-#. Make sure :ref:`THR_FS_VALUE<THR_FS_VALUE>` is an adequate number to trigger the
-   failsafe function on the autopilot.
-#. Make sure :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>` or :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` , or both are enabled (set to a non-zero value).
-#. Connect on the mission planner with your RC transmitter on. Verify on
-   the bottom right corner of the HUD that you are “flying” in a non
-   auto mode (Manual, Stabilize, FBW are ok).
-#. Turn off your transmitter. After :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` sec , if enabled, the flight mode should
-   switch to :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>`. After :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` sec, if enabled, the flight mode should switch to :ref:`FS_LONG_ACTN<FS_LONG_ACTN>`.
-   If you observe this behavior, your failsafe function has been set up
-   correctly.
+Some very old RC receivers cannot be set to send "no pulses" when losing RC signal and simple hold the ROLL/PITCH/YAW RC channels at their last value and set the throttle channel to its minimum PWM value (low throttle). For those, the only way to setup an RC failsafe is to set the :ref:`THR_FS_VALUE<THR_FS_VALUE>` to slightly above that value and use the transmitters trim tab to raise the idle stick value 40-50us above that for normal operation.
```

Comment:
There is a video from Randy that was very helpful to get me to understand what needed to be done when I first was setting this up, youtube:: qf8YinLKQww  might be worthwhile to link here, even though its old tech ?

Reply:
I am reticent to show more info on a method we now highly discourage....I think having the bare minimum is sufficient...I debated even including this section...